### ISSUE TITLE
Allow newer sqlmodel versions

### ISSUE THREAD

**Issue by vtananko** [redb0/fastapi-logger#5]
### Problem
```sh
$ poetry add fastapi-structlog
Using version ^0.4.1 for fastapi-structlog

Updating dependencies
Resolving dependencies... (0.0s)

Because no versions of fastapi-structlog match >0.4.1,<0.5.0
 and fastapi-structlog (0.4.1) depends on sqlmodel (>=0.0.19,<0.0.20), fastapi-structlog (>=0.4.1,<0.5.0) requires sqlmodel (>=0.0.19,<0.0.20).
So, because omega-core depends on both sqlmodel (^0.0.21) and fastapi-structlog (^0.4.1), version solving failed.
```
From [Poetry Dependency specification](https://python-poetry.org/docs/dependency-specification/#caret-requirements)
> Caret requirements allow [SemVer](https://semver.org/) compatible updates to a specified version. An update is allowed if the new version number does not modify the left-most non-zero digit in the major, minor, patch grouping. For instance, if we previously ran poetry add requests@^2.13.0 and wanted to update the library and ran poetry update requests, poetry would update us to version 2.14.0 if it was available, but would not update us to 3.0.0. If instead we had specified the version string as ^0.1.13, poetry would update to 0.1.14 but not 0.2.0. **0.0.x is not considered compatible with any other version.**

### Solution

```diff
-- a/pyproject.toml
+++ b/pyproject.toml
@@ -19,7 +19,7 @@ rich = "^13.7.1"
 pydantic-settings = "^2.3.4"
 asgi-correlation-id = "^4.3.1"
 sentry-sdk = "^2.9.0"
-sqlmodel = "^0.0.19"
+sqlmodel = ">=0.0.19,<0.1.0"
 
 
 [tool.poetry.group.linter.dependencies]
```

**Comment by redb0** [comment]
Thank you for participating

**Comment by redb0** [comment]
Resolved in the merge request #6
